ui/src/main/webapp/home/index.js

```diff
@@ -18,61 +15,12 @@ import Divider from 'material-ui/Divider'
 import { cyan500 } from 'material-ui/styles/colors'
 import RaisedButton from 'material-ui/RaisedButton'
 import MapDisplay from 'components/MapDisplay'
+import Spinner from 'components/Spinner'
 
+import * as selectors from './reducer'
```

Comment:
I think this is an okay use case for `*` imports because he is using everything the module is exporting and both files are tightly coupled.